documentation/docs/src/user-guide/ibc.md

```diff
@@ -0,0 +1,137 @@
+# Using IBC with Namada
+
+This document describes using the inter-blockchain communication (IBC) protocol with Namada. This documentation covers being able create connections through IBC as well as setting up local fractal instances of Namada for testing purposes.
```

Comment:
should cover create clients, connections, and channels, then sending transfers

also we should just say "local instances" of Namada

Reply:
(hermes may do this automatically)

Reply:
Yes I like this structure, will ammend

Reply:
Actually, I just realised I'm not sure what you mean by creating clients here

Reply:
creating IBC clients (light clients), IBC connections, and IBC channels. Maybe `hermes start` does this?